hstest/testing/execution/process/cpp_executor.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from hstest.common.os_utils import is_windows
+from hstest.testing.execution.process_executor import ProcessExecutor
+from hstest.testing.execution.searcher.cpp_searcher import CppSearcher
+
+
+class CppExecutor(ProcessExecutor):
+    def __init__(self, source_name: str = None):
+        super().__init__(CppSearcher().find(source_name))
+
+        self.without_extension = os.path.splitext(self.runnable.file)[0]
+
+        if is_windows():
+            self.executable = self.without_extension
+            self.file_name = self.executable + '.exe'
+        else:
+            self.executable = f'./{self.without_extension}'
+            self.file_name = self.without_extension
+
+    def _compilation_command(self):
+        return ['g++', '-o', self.file_name, self.runnable.file]
```

Comment:
```suggestion
        return ['g++', '-std', 'c++20', '-pipe', '-O2', '-static', '-o', self.file_name, self.runnable.file]
```

https://github.com/hyperskill/submissions-checker/blob/c98e3c8026029945d0664335470fd1213fdb88da/stepic_plugins/settings/base.py#L319

I haven't tested it. We need to check that this works